src/main/java/com/ericsson/ei/jmespath/JmesPathInterface.java

```diff
@@ -72,8 +72,10 @@ public JsonNode runRuleOnEvent(String rule, String event) {
 
         try {
             Expression<JsonNode> expression = jmespath.compile(rule);
+            LOGGER.debug("EXPRESSION : {}", expression );
```

Comment:
These 2 log statements could be merged into one, so it's easier to follow.

Reply:
ok